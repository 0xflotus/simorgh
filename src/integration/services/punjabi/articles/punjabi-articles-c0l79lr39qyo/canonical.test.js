/**
 * @service punjabi
 * @pathname /punjabi/articles/c0l79lr39qyo
 */

import runCanonicalTests from '../../../../pages/articles/canonicalTests';

describe('Canonical punjabi articles', runCanonicalTests);
