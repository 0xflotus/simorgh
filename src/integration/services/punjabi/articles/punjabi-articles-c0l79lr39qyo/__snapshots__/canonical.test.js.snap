// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical punjabi articles A11y I can see the skip to content link 1`] = `"ਸਮੱਗਰੀ 'ਤੇ ਜਾਓ"`;

exports[`Canonical punjabi articles Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=73&amp;p=punjabi.articles.c0l79lr39qyo.page&amp;x1=[urn:bbc:optimo:c0l79lr39qyo]&amp;x2=[responsive]&amp;x3=[news-punjabi]&amp;x4=[pa]&amp;x7=[article]&amp;x8=[simorgh-nojs]&amp;x9=[ਕੰਨ+ਫੜਕੇ+ਧੁੱਪ+&#x27;ਚ+ਉੱਠਕ+ਬੈਠਕ+:+ਬੱਚੇ+ਕਹਿੰਦੇ+ਸਜ਼ਾ+ਵਾਂਗ+ਹੈ+ਤੇ+ਸਕੂਲ+ਵਾਲੇ+ਕਹਿੰਦੇ+&#x27;ਦਿਮਾਗ+ਤੇਜ਼&#x27;+ਕਰਨ+ਵਾਲਾ+ਯੋਗਾ]&amp;x11=[2019-07-12T14:47:57.468Z]&amp;x12=[2019-07-12T14:47:57.468Z]\\"/>"`;

exports[`Canonical punjabi articles Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/punjabi"`;

exports[`Canonical punjabi articles Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. ਬਾਹਰੀ ਸਾਈਟਾਂ ਦੀ ਸਮਗਰੀ ਲਈ ਬੀਬੀਸੀ ਜ਼ਿੰਮੇਵਾਰ ਨਹੀਂ ਹੈ ਬਾਹਰੀ ਲਿੰਕਿੰਗ ਲਈ ਸਾਡੇ ਤਰੀਕੇ ਬਾਰੇ ਪੜ੍ਹੋ"`;

exports[`Canonical punjabi articles Header I can see the branding 1`] = `"BBC News, ਪੰਜਾਬੀ"`;

exports[`Canonical punjabi articles I can see an image with a caption 1`] = `"https://ichef.bbci.co.uk/news/640/cpsdevpb/4d4a/test/5143a9a0-fcc0-11e9-9afe-1371928331c5.jpg"`;

exports[`Canonical punjabi articles I can see an image with a caption 2`] = `"ਤਸਵੀਰ ਕੈਪਸ਼ਨ, ਇਹ ਪਰੀਖਿਆ ਚਿੱਤਰ, ਕਾਪੀਰਾਈਟ ਬੀਬੀਸੀ, ਫਰਾਂਸ ਦਾ ਨਕਸ਼ਾ ਦਰਸਾਉਂਦਾ ਹੈ. ਚਿੱਤਰ ਪਹਿਲੇ ਤਿੰਨ ਬਲਾਕਾਂ ਵਿੱਚ ਹੈ ਅਤੇ ਇਸਦਾ ਸਿਰਲੇਖ ਹੈ."`;

exports[`Canonical punjabi articles SEO Canonical link 1`] = `"http://localhost:7080/punjabi/articles/c0l79lr39qyo"`;

exports[`Canonical punjabi articles SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical punjabi articles SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical punjabi articles SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical punjabi articles SEO Heading level 1 1`] = `"ਇਹ ਇਸ ਪਰੀਖਿਆ ਲੇਖ ਦਾ ਸਿਰਲੇਖ ਹੈ"`;

exports[`Canonical punjabi articles SEO Lang attribute 1`] = `"pa"`;

exports[`Canonical punjabi articles SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/punjabi.png",
      "width": 1024,
    },
    "name": "BBC News ਪੰਜਾਬੀ",
    "noBylinesPolicy": "https://www.bbc.com/punjabi/institutional-49282853#authorexpertise",
  },
  "dateModified": "2019-07-12T14:47:57.468Z",
  "datePublished": "2019-07-12T14:47:57.468Z",
  "headline": "ਕੰਨ ਫੜਕੇ ਧੁੱਪ 'ਚ ਉੱਠਕ ਬੈਠਕ : ਬੱਚੇ ਕਹਿੰਦੇ ਸਜ਼ਾ ਵਾਂਗ ਹੈ ਤੇ ਸਕੂਲ ਵਾਲੇ ਕਹਿੰਦੇ 'ਦਿਮਾਗ ਤੇਜ਼' ਕਰਨ ਵਾਲਾ ਯੋਗਾ",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/punjabi.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/punjabi/articles/c0l79lr39qyo",
    "@type": "WebPage",
    "name": "ਕੰਨ ਫੜਕੇ ਧੁੱਪ 'ਚ ਉੱਠਕ ਬੈਠਕ : ਬੱਚੇ ਕਹਿੰਦੇ ਸਜ਼ਾ ਵਾਂਗ ਹੈ ਤੇ ਸਕੂਲ ਵਾਲੇ ਕਹਿੰਦੇ 'ਦਿਮਾਗ ਤੇਜ਼' ਕਰਨ ਵਾਲਾ ਯੋਗਾ",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/punjabi.png",
      "width": 1024,
    },
    "name": "BBC News ਪੰਜਾਬੀ",
    "publishingPrinciples": "https://www.bbc.com/punjabi/institutional-49282853",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/punjabi.png",
  "url": "http://localhost:7080/punjabi/articles/c0l79lr39qyo",
}
`;

exports[`Canonical punjabi articles SEO OG description 1`] = `"ਕੰਨ ਫੜਕੇ ਧੁੱਪ 'ਚ ਉੱਠਕ ਬੈਠਕ : ਬੱਚੇ ਕਹਿੰਦੇ ਸਜ਼ਾ ਵਾਂਗ ਹੈ ਤੇ ਸਕੂਲ ਵਾਲੇ ਕਹਿੰਦੇ 'ਦਿਮਾਗ ਤੇਜ਼' ਕਰਨ ਵਾਲਾ ਯੋਗਾ"`;

exports[`Canonical punjabi articles SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/punjabi.png"`;

exports[`Canonical punjabi articles SEO OG image alt 1`] = `"BBC News ਪੰਜਾਬੀ"`;

exports[`Canonical punjabi articles SEO OG locale 1`] = `"pa-IN"`;

exports[`Canonical punjabi articles SEO OG site name 1`] = `"BBC News ਪੰਜਾਬੀ"`;

exports[`Canonical punjabi articles SEO OG title 1`] = `"ਕੰਨ ਫੜਕੇ ਧੁੱਪ 'ਚ ਉੱਠਕ ਬੈਠਕ : ਬੱਚੇ ਕਹਿੰਦੇ ਸਜ਼ਾ ਵਾਂਗ ਹੈ ਤੇ ਸਕੂਲ ਵਾਲੇ ਕਹਿੰਦੇ 'ਦਿਮਾਗ ਤੇਜ਼' ਕਰਨ ਵਾਲਾ ਯੋਗਾ - BBC News ਪੰਜਾਬੀ"`;

exports[`Canonical punjabi articles SEO OG type 1`] = `"article"`;

exports[`Canonical punjabi articles SEO OG url 1`] = `"http://localhost:7080/punjabi/articles/c0l79lr39qyo"`;

exports[`Canonical punjabi articles SEO Page title 1`] = `"ਕੰਨ ਫੜਕੇ ਧੁੱਪ 'ਚ ਉੱਠਕ ਬੈਠਕ : ਬੱਚੇ ਕਹਿੰਦੇ ਸਜ਼ਾ ਵਾਂਗ ਹੈ ਤੇ ਸਕੂਲ ਵਾਲੇ ਕਹਿੰਦੇ 'ਦਿਮਾਗ ਤੇਜ਼' ਕਰਨ ਵਾਲਾ ਯੋਗਾ - BBC News ਪੰਜਾਬੀ"`;

exports[`Canonical punjabi articles SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical punjabi articles SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical punjabi articles SEO Twitter creator 1`] = `"@bbcnewspunjabi"`;

exports[`Canonical punjabi articles SEO Twitter description 1`] = `"ਕੰਨ ਫੜਕੇ ਧੁੱਪ 'ਚ ਉੱਠਕ ਬੈਠਕ : ਬੱਚੇ ਕਹਿੰਦੇ ਸਜ਼ਾ ਵਾਂਗ ਹੈ ਤੇ ਸਕੂਲ ਵਾਲੇ ਕਹਿੰਦੇ 'ਦਿਮਾਗ ਤੇਜ਼' ਕਰਨ ਵਾਲਾ ਯੋਗਾ"`;

exports[`Canonical punjabi articles SEO Twitter image alt 1`] = `"BBC News ਪੰਜਾਬੀ"`;

exports[`Canonical punjabi articles SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/punjabi.png"`;

exports[`Canonical punjabi articles SEO Twitter site 1`] = `"@bbcnewspunjabi"`;

exports[`Canonical punjabi articles SEO Twitter title 1`] = `"ਕੰਨ ਫੜਕੇ ਧੁੱਪ 'ਚ ਉੱਠਕ ਬੈਠਕ : ਬੱਚੇ ਕਹਿੰਦੇ ਸਜ਼ਾ ਵਾਂਗ ਹੈ ਤੇ ਸਕੂਲ ਵਾਲੇ ਕਹਿੰਦੇ 'ਦਿਮਾਗ ਤੇਜ਼' ਕਰਨ ਵਾਲਾ ਯੋਗਾ - BBC News ਪੰਜਾਬੀ"`;
